### PR TITLE
Update with copy column

### PR DESCRIPTION
**Pull request by abellgithub**
There are several drivers for which I couldn't locate the code:
DB2
Geopackage

Leveller claimed not to support creation, but the code says otherwise.
TerraSAR claimed to support creation, but the code says otherwise.
DTED looks as if there is a function to do creation in addition to copy, but it doesn't seem to be registered as a callback.

I'm also wondering about GDAL_DCAP_CREATECOPY and GDAL_DCAP_CREATE, as they only seem to be used in the NITF driver.  Should they be removed or flushed out?

### CODE REVIEW

**Comment by rouault** [comment]
* ogr/ogrsf_frmts/db2
* ogr/ogrsf_frmts/gpkg
* DTED: DTEDCreate() has not the interface expected by pfnCreate. It is quite specialized
* re NITF: GDAL_DCAP_CREATECOPY and GDAL_DCAP_CREATE are needed by a number of things, in particular the utilities, so please keep them :-)

File: gdal/frmts/formats_list.html [hunk 1]
```diff
@@ -14,13 +14,15 @@
 <th>Long Format Name</th>
 <th>Code</th>
 <th>Creation</th>
+<th>Copy</th>
```
Comment (rouault) [inline gdal/frmts/formats_list.html]: Perhaps as a footnote we should define the terms:
Creation = Creation of a new dataset from scratch
Copy = Copy from an existing source dataset

**Comment by abellgithub** [comment]
Didn't see that DCAP_CREATE/CREATECOPY were usually set by the driver manager.
Updated as suggested.

**Comment by rouault** [comment]
Committed in r39960. Thanks

**Pull request closed without merging**